### PR TITLE
Fix/cleanup readme

### LINKED ISSUE #186: Refresh wordpress.org image assets
Docs: https://developer.wordpress.org/plugins/wordpress-org/plugin-assets/

Should all be saved to `.wordpress-org` directory, as the workflow from #175 will look there for them by default (and push to the `assets` directory), and it has an equivalence in the existing `.github` directory.

These will then appear on https://wordpress.org/plugins/wp-parsely/ and localized variants (e.g. https://ar.wordpress.org/plugins/wp-parsely/)

See https://wordpress.org/plugins/wordpress-seo/ and https://ar.wordpress.org/plugins/wordpress-seo/ as an example of the banners.

## Screenshots

Of the four screenshots, 3 of them are snippets of code.

It would be nice to include screenshot of the widget form, and refreshed settings page, and one of the Parse.ly dashboard show where the data will end up. As more features are added to the plugin (new widgets, stats areas), then these should be added as screenshots as well.

There are no specific sizes needed, but the current ones are all different sizes, and could be framed better.

## Banners
The following can be added:

- Normal Banner: banner-772x250.(jpg|png)
- Normal Banner (RTL): banner-772x250-rtl.(jpg|png)
- High-DPI (Retina): banner-1544x500.(jpg|png)
- High-DPI (Retina RTL): banner-1544x500-rtl.(jpg|png)

The RTL versions are only needed if the LTR banner doesn't look good for RTL locales.

The High-DPI versions don't work on their own - the Normal banner needs to be present as well.

## Plugin Icons

The following can be added:

- Normal: icon-128x128.(png|jpg)
- High-DPI (Retina): icon-256x256.(png|jpg)
- SVG: icon.svg

Existing icons, grabbed from https://plugins.svn.wordpress.org/wp-parsely/assets/:

icon-128x128
icon-256x256

### PR DESCRIPTION
**Pull request by mikeyarce**
Partially Fixes: https://github.com/Parsely/wp-parsely/issues/186

I think the main remaining task is to add a plugin banner cc @dabowman

What this does:
- Updates screenshots
- Moves from code screenshots to code snippets
- Improves installation instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,14 +28,22 @@ Feedback, suggestions, questions or concerns? E-mail us at [support@parsely.com]
...
 
-1. This plug-in requires an active version of Parse.ly. We offer a free trial, [sign up here](http://www.parsely.com/trial/?utm_medium=referral&utm_source=wordpress.org&utm_content=wp-parsely)
-1. If you haven't already done so, [sign up for a trial of Parse.ly](http://www.parsely.com/trial/?utm_medium=referral&utm_source=wordpress.org&utm_content=wp-parsely)
-1. Download the plugin
-1. Upload the entire `wp-parsely` folder to your `/wp-content/plugins` directory
-1. Activate the plugin through the 'Plugins' menu in WordPress (look for "Parse.ly")
-1. Head to the settings page for the plugin (should be /wp-admin/options-general.php?page=parsely)
-1. Set your Site ID, which is your own site domain name (e.g., `mysite.com`)
-1. Save your changes and enjoy your data!
+The WP-Parsely plugin requires an active Parse.ly account. Parse.ly gives creators, marketers and developers the tools to understand content performance, prove content value, and deliver tailored content experiences that drive meaningful results.
```
Comment (GaryJones) [inline README.md]:
```suggestion
The plugin requires an active Parse.ly account. Parse.ly gives creators, marketers, and developers the tools to understand content performance, prove content value, and deliver tailored content experiences that drive meaningful results.
```

File: README.md [hunk 2]
```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
 
 ## Screenshots
 
-1. The main settings screen of the wp-parsely plugin  
+1. The main settings screen of the WP-Parsely plugin
```
Comment (GaryJones) [inline README.md]: The plugin isn't called `WP-Parsely`. It's called `Parse.ly`, and has a slug of `wp-parsely`. Let's aim not to mix the two.
```suggestion
1. The main settings screen of the Parse.ly plugin.  
```

File: README.md [hunk 3]
```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
...
+
+## Sample Parse.ly metadata
+
+The standard Parse.ly JavaScript tracker inserted before the closing `body` tag:
+
+    <!-- START Parse.ly Include: Standard -->
+
+       <script data-cfasync="false" id="parsely-cfg" data-parsely-site="wpvip.com" src="//cdn.parsely.com/keys/wpvip.com/p.js"></script>
+
+    <!-- END Parse.ly Include: Standard -->
```
Comment (GaryJones) [inline README.md]: This bit isn't the metadata, so it should be moved outside of the `Sample Parse.ly metadata` section (can have its own heading).

Also:

```suggestion
The standard Parse.ly JavaScript tracker inserted before the closing `body` tag:

    <!-- START Parse.ly Include: Standard -->

       <script data-cfasync="false" id="parsely-cfg" data-parsely-site="example.com" src="https://cdn.parsely.com/keys/example.com/p.js"></script>

    <!-- END Parse.ly Include: Standard -->
```

File: README.md [hunk 4]
```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
...
+
+A sample `JSON-LD` meta tag for a home page or section page:
```
Comment (GaryJones) [inline README.md]:
```suggestion
A sample `JSON-LD` meta tag and structured data for a home page or section page:
```

Really, the metatag with the plugin version is separate from the JSON-LD. We may be able to remove it.

File: README.md [hunk 5]
```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
...
+    <!-- BEGIN wp-parsely Plugin Version 2.5.0 -->
+        <meta name="wp-parsely_version" id="wp-parsely_version" content="2.3"/>
+            <script type="application/ld+json">
+                {"@context":"http:\/\/schema.org","@type":"WebPage","headline":"WordPress VIP","url":"http:\/\/wpvip.com\/"}
+            </script>
+    <!-- END wp-parsely Plugin Version 2.5.0 -->
 
-3. A sample `JSON-LD` meta tag for a home page or section page
-![3. A sample `JSON-LD` meta tag for a home page or section page](.wordpress-org/screenshot-3.png)
+A sample `JSON-LD` meta tag for an article or post:
```
Comment (GaryJones) [inline README.md]:
```suggestion
A sample `JSON-LD` structured data element for an article or post:
```

**Review by GaryJones — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mikeyarce) [reply README.md]: This was taken from the main Parse.ly homepage - but I agree this comma is needed.

**Review by GaryJones — APPROVED** [review]

**Pull request merged**